code/__HELPERS/_logging.dm

```diff
@@ -58,6 +58,10 @@
 	if (CONFIG_GET(flag/log_game))
 		WRITE_LOG(GLOB.world_game_log, "GAME: [text]")
 
+/proc/log_objective(whom, objective, admin_involved)
```

Comment:
Is it just me or did you forget to actually add `[objective]`

Reply:
Indeed. This is why I don't code while half-asleep.